src/utils.ts

```diff
@@ -49,3 +49,13 @@ export function sudo(cmd: string): Promise<string | null> {
     });
   });
 }
+
+export function isDomainInHostFile(hostFileContents: string, domain: string): boolean {
+  // Do a check for a full match since a string includes can be fooled by 
+  // a subdomain being present in the host file.
+  const isPresent = hostFileContents
+    .replace(/\s+/g, " ")
+    .split(" ")
+    .filter(item => item === domain).length > 0;
+ return isPresent;
+}
```

Comment:
This looks like pretty solid code, but I'm not 100% sure it'll work for all possible hostfile contents, such as tab characters, different encodings, or comments. Maybe it would be easier to bring in a tool like https://www.npmjs.com/package/hostile which has a well-tested hostfile parser.

Reply:
This was a pretty quick and dirty way of improving the checking. Your are right that can be fooled by domains in comments, which the previous code had a problem with as well. I'll take a look hostile and see what that looks like to bring in.